tests/src/__init__.py

```diff
@@ -0,0 +1,4 @@
+from build import setup_jinja_environment
+
+alerts = []
```

Comment:
Minor: rather than hard-code these here, we could maintain the convenience of a no-alerts environment by using a fixture instead.

```
# in tests/conftest.py
@pytest.fixture()
def env():
  return setup_jinja_environment([])
```

In individual tests, we can choose whether we want to add alerts.

```
def test_current_alerts_page(alert_dict, env):
  env.globals['alerts'] = [alert_dict]
  template = env.get_template("src/current-alerts.html")
  content = template.render()
  html = BeautifulSoup(content, 'html.parser')
  ...

def test_individual_alert_page(alert_dict, env):
  template = env.get_template("src/alert.html", {'alert_data': alert_dict})
  ...
```